content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1177,10 +1180,10 @@ Setting `fieldPath` is optional.
 
 #### The `optionalOldSelf` field {#field-optional-oldself}
 
-{{< feature-state state="alpha" for_k8s_version="v1.29" >}}
+{{< feature-state state="beta" for_k8s_version="v1.30" >}}
 
 The feature [CRDValidationRatcheting](#validation-ratcheting) must be enabled in order to 
-make use of this field.
+make use of this field. If the feature is not enabled, this field is ignored.
```

Comment:
This is what I expected to see:
```suggestion
If your cluster does not have [CRD validation ratcheting](#validation-ratcheting) enabled, 
the CustomResourceDefinition API doesn't include this field, and trying to set it may result
in an error.
```